Maths/fibonacci.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * Run script and test execution time with following script
+        $executionTime = New ExecutionTime();
+        print_r(fibonacciRecursive(10));
+ */
+
+class ExecutionTime
+{
+    private $start_time = 0;
+    private $end_time = 0;
+    private $execution_time = 0;
+
+    public function __construct()
+    {
+        $this->start_time = microtime(true);
+    }
+
+    public function __destruct()
+    {
+        $this->end_time = microtime(true);
+        $this->execution_time = $this->end_time - $this->start_time;
+        echo "Executed in $this->execution_time seconds\n";
+    }
+}
+
+function fibonacciRecursive(int $num)
```

Comment:
Can we improve this function a bit?  It's not technically a recursive function and can be simplified quite a bit.  Something like this would be simpler

```php
function fibonacciRecursive(int $num)
{
    if ($num < 0) {
        throw new \Exception("Number must be greater than 0");
    }

    if ($num == 0) {
        return 0;
    } else if ($num == 1) {
        return 1;
    } else {
        return fibonacciRecursive($num-1) + fibonacciRecursive($num-2);
}
```